app/Base.lproj/About.storyboard

```diff
@@ -696,11 +696,28 @@
                                     </subviews>
                                 </tableViewCellContentView>
                             </tableViewCell>
+                            <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" reuseIdentifier="Status Bar" textLabel="RGb-hU-8nK" style="IBUITableViewCellStyleDefault" id="n5A-k0-80S" userLabel="Status Bar">
```

Comment:
This probably wants a better label, such as "Hide Status Bar"

Reply:
User Label is what is shown in the editor, not what is displayed on the device. I have "Show Status Bar" as the label in the enclosed content view.